src/cpu/o3/fetch.cc

```diff
@@ -193,7 +193,13 @@ Fetch::FetchStatGroup::FetchStatGroup(CPU *cpu, Fetch *fetch)
              "Number of instructions fetched each cycle (Total)"),
     ADD_STAT(idleRate, statistics::units::Ratio::get(),
              "Ratio of cycles fetch was idle",
-             idleCycles / cpu->baseStats.numCycles)
+             idleCycles / cpu->baseStats.numCycles),
+    ADD_STAT(fetchBubbles, statistics::units::Count::get(),
+             "Stat for Top-Down Methodology, number of instructions not "
```

Comment:
Could be repreased like "number of empty slots where no uop was delivered to decode"